ci-operator/step-registry/openshift/microshift/rebase/openshift-microshift-rebase-ref.yaml

```diff
@@ -1,6 +1,6 @@
 ref:
   as: openshift-microshift-rebase
-  from: rebase-payload
+  from: src
```

Comment:
Is `rhel-9-release-golang-1.20-openshift-4.14` base for `src`?
I'm thinking ahead when we bump that and perhaps some day common code in `commands.sh` will need some tweaking (or some python libraries get updated so our code will work with older but not with newer), but we can deal with that in (far?) future

Reply:
Yes, `src` image comes from `root` image, which in turn comes from `build_root`, which is updated everytime we get a new go version (or ocp version, for that matter).